src/components/about.tsx

```diff
@@ -0,0 +1,17 @@
+interface AboutProps {
+  name: string;
+  description: string;
+}
```

Comment:
you're passing in props here, but we actually don't need props passed in here haha should've specified in the issue description. we'll have some actual text on the figma there soon. for now you can just put the lorem ipsum text in